layers/meta-balena-jetson/recipes-bsp/tegra-binaries/tegra194-flash-dry_32.2.0.bb

```diff
@@ -132,6 +132,7 @@ do_configure() {
 
     mkdir -p ${DEPLOY_DIR_IMAGE}/bootfiles
     cp ./cboot_t194.bin ${DEPLOY_DIR_IMAGE}/bootfiles/
+    echo "0" > ${DEPLOY_DIR_IMAGE}/bootfiles/none.bin
```

Comment:
What is this for?

Reply:
Is it the padding?

Reply:
Thanks @floion , I have updated the PR to address the requested changes.